demos/simple_jump_bot/README.md

```diff
@@ -0,0 +1,22 @@
+# HammettSimpleJumpBot
+
+This demo represents the `jump` mechanism which Hammett provides. This mechanism allows sending yours screens as new message and saving the previous ones in a chat history.
+
+To use it, you need to create a keyboard with a button which source type is `JUMP_SOURCE_TYPE` and a source is a screen which you want to be sent as new message. Well, you can see the implementation in the `demo.py` module.
+
+To watch the result you need to set the `HAMMETT_SETTINGS_MODULE` and `TOKEN` environment variables and run the `demo.py` script. You can use the following command:
+
+```bash
+env PYTHONPATH=$(pwd) HAMMETT_SETTINGS_MODULE=demos.simple_jump_bot.settings TOKEN=your-token python3 demos/simple_jump_bot/demo.py
+```
```

Comment:
Make it possible to load the values from `.env`. Describe it as the second option.